broker/src/tunneldigger_broker/broker.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, hook_manager, max_tunnels, tunnel_id_base, tunnel_port_base,
         self.hook_manager = hook_manager
         self.max_tunnels = max_tunnels
         self.tunnel_id_base = tunnel_id_base
-        self.tunnel_ids = set(xrange(tunnel_id_base, tunnel_id_base + max_tunnels + 1))
+        self.tunnel_ids = set(xrange(tunnel_id_base, tunnel_id_base + max_tunnels))
```

Comment:
While we are changing this, can we switch to `range` instead?

Reply:
But `xrange` is better because it doesn't allocate a list, right?

Reply:
Yes, it just returns a generator. But in Python 3 it got removed as `range` returns a generator there. But perhaps we should just leave it for now and handle it in a later "Python 3 support" PR.

Reply:
I think it'd make more sense to just at some point (the sooner the better :D ) move to Python 3 and drop support for Python 2.

But, if you want, I can also do `s/xrange/range/` here and now.